src/main/java/dukeApp/duke/Duke.java

```diff
@@ -0,0 +1,50 @@
+package dukeApp.duke;
+import dukeApp.files.*;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Duke {
+    private TaskList tasks;
+    private Ui ui;
+    private Storage storage;
+
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (FileNotFoundException e) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * Run the app by reading user commands and writing to file to save changes
+<<<<<<< HEAD
+     * 
+=======
+>>>>>>> branch-A-CodingStandard
+     */
+    public void run() {
+        ui.input(tasks);
+        try {
+            storage.appendToFile(tasks.aList);
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " +e.getMessage());
+        }
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
```

Comment:
Perhaps you could consider making the logo a FINAL variable as it won't change? Other than that your code looks good!